components/dropdown/dropdown-menu.js

```diff
@@ -91,8 +91,15 @@ class DropdownMenu extends DropdownContentMixin(LitElement) {
 	}
 
 	__getMenuElement() {
-		return this.shadowRoot.querySelector('.d2l-dropdown-content-container > slot')
-			.assignedNodes()
+		let slot = this.shadowRoot.querySelector('.d2l-dropdown-content-container > slot');
```

Comment:
Same comment here as in the other pull request. I think this can be solved by simplifying the selector to not reference the `slot` element, since it doesn't exist on browsers who don't support shadow DOM.